### PR TITLE
Add README.md for examples

### PR DESCRIPTION
**Pull request by v-rudkovskiy**

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Examples
+
+## Project setup
+```
+composer install
```
Comment (kkorotkov) [inline examples/README.md:7]: Should this be executed from examples directory or project root?

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+```
+
+### Run example as phpunit test
+```
+php vendor/phpunit/phpunit/phpunit examples
```
Comment (kkorotkov) [inline examples/README.md:12]: Same here: from examples subdir or project root?

**Comment by kkorotkov** [comment]
Ensure branch has no conflicts and can be rebased to the target.

**Comment by kkorotkov** [comment]
@v-rudkovskiy Any news here?

File: examples/README.md (hunk 1 above)
Comment (v-rudkovskiy) [reply examples/README.md:7]: We need to be at the root of the project. Added clarification in README.md

File: examples/README.md (hunk 2 above)
Comment (v-rudkovskiy) [reply examples/README.md:12]: Added clarification in README.md

**Comment by v-rudkovskiy** [comment]
> 
> 
> Ensure branch has no conflicts and can be rebased to the target.

Everything is fine, there are no conflicts

**Comment by kkorotkov** [comment]
image

**Review by kkorotkov — CHANGES_REQUESTED** [review]
See the screenshot above

**Pull request merged**